### PR TITLE
Add "Hide Already Selected" example for Select component

### PR DESCRIPTION
**Pull request by SergeyVolynkin**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your pull request, thank you!

* [x] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a pull request to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you pull request.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

File: components/select/demo/hide-selected.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+order: 22
+title:
+  zh-CN: 已隐藏已选中
```
Comment (ztplz) [inline components/select/demo/hide-selected.md]: Just a little comment.  `隐藏已选择选项` is better than `已隐藏已选中`.

**Review by ztplz — APPROVED** [review]

**Comment by ztplz** [comment]
LGTM. Thank you.

File: components/select/demo/hide-selected.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    const filteredOptions = OPTIONS.filter(o => !selectedItems.includes(o));
+    return (
+      <Select
+        mode="multiple"
+        placeholder="Inserted are removed"
+        value={selectedItems}
+        onChange={this.handleChange}
+        style={{ width: '100%' }}
+      >
+        {filteredOptions.map(o => (
```
Comment (afc163) [inline components/select/demo/hide-selected.md]: `o` => `item`

**Comment by zy410419243** [comment]
Need to update snapshots.

**Comment by SergeyVolynkin** [comment]
@ztplz @afc163 Thank you for your comments, made the changes.

**Comment by SergeyVolynkin** [comment]
@zy410419243 
snapshots updated

**Pull request merged**